lib/features/dragging/Dragging.js

```diff
@@ -282,7 +282,7 @@ export default function Dragging(eventBus, canvas, selection, elementRegistry) {
 
   function checkCancel(event) {
 
-    if (event.which === 27) {
+    if (event.code === 'Escape') {
```

Comment:
This will break the feature for IE entirely: https://caniuse.com/keyboardevent-code